test/bats/helpers/lxly-bridge.bash

```diff
@@ -356,3 +357,59 @@ function wait_for_expected_token() {
         sleep "$poll_frequency"
     done
 }
+
+function get_claim() {
+    local l1_rpc_network_id="$1"
```

Comment:
Rename this, as it is not going to be an L1 network always, but it also can be an L2 one.
```suggestion
    local network_id="$1"
```